mm.c

```diff
@@ -863,6 +949,11 @@ static void test_find_free_block() {
 	assert(GET_SIZE(bp) == 53);
 	printf("Case 7 passed.\n");
 
+	// Case 8: find zero size - should still attempt to get the next-largest block
+	bp = find_free_block(0);
```

Comment:
our malloc should have the same behavior as the std malloc, thus malloc(0) should return NULL though